src/plugins/content/accessibility/keyNavigation/strategies/itemNavigation.js

```diff
@@ -43,75 +43,126 @@ export default {
+         * @param {jQuery} group - The group container
+         * @param {Number|Function} [defaultPosition=0] - the default position the group should set the focus on
+         * @returns {keyNavigator} - the created navigator, if the list of element is not empty
+         */
+        const addNavigator = ($elements, group, defaultPosition = 0) => {
+            const elements = navigableDomElement.createFromDoms($elements);
+            if (elements.length) {
+                const navigator = keyNavigator({
+                    elements,
+                    group,
+                    defaultPosition,
+                    propagateTab: false
+                });
+                this.keyNavigators.push(navigator);
+                return navigator;
+            }
+        };
+
+        /**
+         * Creates and setups a keyNavigator for the interaction inputs.
+         * @param {jQuery} $elements - The list of navigable elements
+         * @param {jQuery} group - The group container
+         * @param {Number|Function} [defaultPosition=0] - the default position the group should set the focus on
+         * @returns {keyNavigator} - The supplied keyNavigator
+         */
+        const addInputsNavigator = ($elements, group, defaultPosition = 0) => {
+            const navigator = addNavigator($elements, group, defaultPosition);
+            if (navigator) {
+                setupItemsNavigator(navigator, config);
+                setupClickableNavigator(navigator);
+
+                // each choice is represented by more than the input, the style must be spread to the actual element
+                navigator
+                    .on('focus', cursor => scrollHelper.scrollTo(
+                        getQtiChoice(cursor).addClass('key-navigation-highlight'),
+                        $content.closest('.content-wrapper')
+                    ))
+                    .on('blur', cursor => getQtiChoice(cursor).removeClass('key-navigation-highlight'));
+            }
+            return navigator;
+        };
+
+        // list the navigable areas inside the item. This could be either the interactions choices or the prompts
+        const $qtiInteractions = $content
             .find('.key-navigation-focusable,.qti-interaction')
-            .filter(function () {
-                //filter out interaction as it will be managed separately
-                return !$(this).parents('.qti-interaction').length;
+            .filter((i, node) => {
+                const $node = $(node);
+                // reduce the list to full interactions, each one will get its own keyNavigator
+                if (!$node.parents('.qti-interaction').length) {
+                    const $choiceInput = $node.find('.qti-choice input');
+                    if ($choiceInput.length) {
+                        // expose the type of choices: checkbox or radio
+                        $node.attr('data-choice-type', $choiceInput.attr('type'));
```

Comment:
Filter method should accept pure function as a callback. I suggest to use `each` for applying side-effects